src/components/snapshots/takeSnapshot.ts

```diff
@@ -130,15 +130,12 @@ const getOptions = async (
 }
 
 const zipSnapshot = async (snapshotFolder: string, snapshotName: string) => {
-  const zip = new zipper()
-  zip.addLocalFolder(snapshotFolder)
-
-  await new Promise((resolve, reject) =>
-    zip.writeZip(path.join(SNAPSHOT_FOLDER, `${snapshotName}.zip`), (error) => {
-      if (error) return reject(error)
-      resolve('done')
-    })
-  )
+  const output = fs.createWriteStream(path.join(SNAPSHOT_FOLDER, `${snapshotName}.zip`))
+  const archive = archiver('zip')
+
+  archive.pipe(output)
+  archive.directory(snapshotFolder, false)
+  archive.finalize()
```

Comment:
The standard snapshot export is now using the same "archiver" zip library. We can eventually remove `adm-zip`, altogether, but it's still needed to un-zipping when uploading snapshots. (Need to find a replacement that can handle encrypted zips)